[BUG] render deploy doesn't work

### What happened?

tried to deploy to render and the build failed

### Steps to Reproduce

- click the deploy to render link in the readme
- continue in the render dashboard, deploy
- deploy/build error

### Component

Other

### Environment

render

### Relevant log output

```shell
2026-02-24T08:23:49.602591478Z ==> It looks like we don't have access to your repo, but we'll try to clone it anyway.
2026-02-24T08:23:49.602611859Z ==> Cloning from https://github.com/CaviraOSS/OpenMemory
2026-02-24T08:23:51.109539755Z ==> Checking out commit afc7db127ecca7c214a23729d625f0a6966a07e3 in branch main
2026-02-24T08:23:52.635669903Z ==> Using Node.js version 22.22.0 (default)
2026-02-24T08:23:52.662486934Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version
2026-02-24T08:23:57.027758092Z ==> Running build command 'npm install && npm run build'...
2026-02-24T08:24:06.733856607Z 
2026-02-24T08:24:06.733887438Z added 441 packages, and audited 442 packages in 9s
2026-02-24T08:24:06.733894618Z 
2026-02-24T08:24:06.733909059Z 57 packages are looking for funding
2026-02-24T08:24:06.733913759Z   run `npm fund` for details
2026-02-24T08:24:06.794674807Z 
2026-02-24T08:24:06.794696647Z 30 vulnerabilities (3 moderate, 26 high, 1 critical)
2026-02-24T08:24:06.794702318Z 
2026-02-24T08:24:06.794708118Z To address issues that do not require attention, run:
2026-02-24T08:24:06.794713838Z   npm audit fix
2026-02-24T08:24:06.794719168Z 
2026-02-24T08:24:06.794724688Z To address all issues (including breaking changes), run:
2026-02-24T08:24:06.794731518Z   npm audit fix --force
2026-02-24T08:24:06.794736888Z 
2026-02-24T08:24:06.794741008Z Run `npm audit` for details.
2026-02-24T08:24:06.97266437Z 
2026-02-24T08:24:06.97269024Z > openmemory-js@1.3.3 build
2026-02-24T08:24:06.97269725Z > tsc -p tsconfig.json
2026-02-24T08:24:06.97270272Z 
2026-02-24T08:24:18.689319278Z src/core/db.ts(2,34): error TS7016: Could not find a declaration file for module 'pg'. '/opt/render/project/src/packages/openmemory-js/node_modules/pg/lib/index.js' implicitly has an 'any' type.
2026-02-24T08:24:18.689338869Z   Try `npm i --save-dev @types/pg` if it exists or add a new declaration (.d.ts) file containing `declare module 'pg';`
2026-02-24T08:24:18.689492772Z src/core/migrate.ts(3,22): error TS7016: Could not find a declaration file for module 'pg'. '/opt/render/project/src/packages/openmemory-js/node_modules/pg/lib/index.js' implicitly has an 'any' type.
2026-02-24T08:24:18.689502162Z   Try `npm i --save-dev @types/pg` if it exists or add a new declaration (.d.ts) file containing `declare module 'pg';`
2026-02-24T08:24:18.689533573Z src/ops/extract.ts(7,20): error TS7016: Could not find a declaration file for module 'fluent-ffmpeg'. '/opt/render/project/src/packages/openmemory-js/node_modules/fluent-ffmpeg/index.js' implicitly has an 'any' type.
2026-02-24T08:24:18.689539033Z   Try `npm i --save-dev @types/fluent-ffmpeg` if it exists or add a new declaration (.d.ts) file containing `declare module 'fluent-ffmpeg';`
2026-02-24T08:24:19.318390069Z ==> Build failed 😞
2026-02-24T08:24:19.318407279Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/render.yaml
+++ b/render.yaml
@@ -3,7 +3,7 @@ services:
     name: openmemory
     env: node
     rootDir: packages/openmemory-js
-    buildCommand: npm install && npm run build
+    buildCommand: npm install --include=dev && npm run build
     startCommand: npm start
     plan: free
     autoDeploy: true

